packages/babel-plugin-transform-function-name/test/fixtures/function-name/unicode-encoding/input.js

```diff
@@ -0,0 +1,3 @@
+var o = {
+  "\uD835\uDC9C"() {}
```

Comment:
The new test cases use `shorthand-properties` which is not supported by `function-name`. We can either plug in `transform-shorthand-properties` or just use ObjectProperty instead.

```js
var o = {
  "\uD835\uDC9C": function () {}
}
```

When unicode escapes are supported, the output should be
```js
var o = {
  "\uD835\uDC9C": function 𝒜() {}
};
```

otherwise, the output should be

```js
var o = {
  "\uD835\uDC9C": function () {}
};
```
In this case the plugin does nothing.

Reply:
How can we plug-in `transform-shorthand-properties` in `transform-function-name`?